Add mm39 UI updates

belkassaby opened the pull request.

belkassaby committed:
update to angular 14

belkassaby committed:
update to angular 14

belkassaby committed:
update to angular 14

belkassaby committed:
update ui for mm39

belkassaby committed:
update gitignore

belkassaby committed:
update readme

iimpulse commented on src/app/analysis/search.service.ts:
```diff
@@ -108,18 +105,18 @@ export class SearchService {
     private sendVariantQueryRequest(paramsIn: any, url: string): Observable<any> {
 
-        //console.log(paramsIn)
+        // console.log(paramsIn)
```
can remove

iimpulse commented on src/app/analysis/snps-indels/snp-indel-details/snp-indel-details.component.ts:
```diff
@@ -16,7 +17,7 @@ export class SnpIndelDetailsComponent implements OnInit {
     @ViewChild('transcriptPaginator', {static: true}) transcriptPaginator: MatPaginator;
     @ViewChild(MatSort, {static: true}) transcriptSort: MatSort;
-    //@ViewChild('strainPaginator', {static: true}) strainPaginator: MatPaginator;
+    // @ViewChild('strainPaginator', {static: true}) strainPaginator: MatPaginator;
```
Prolly remove?

iimpulse commented on src/app/analysis/snps-indels/snp-indel-details/snp-indel-details.component.ts:
```diff
@@ -63,13 +64,13 @@ export class SnpIndelDetailsComponent implements OnInit {
 
         this.transcriptPageLength = this.variant.transcripts.length;
-        //this.strainPageLength = this.variant.strains.length;
+        // this.strainPageLength = this.variant.strains.length;
```
Try to tend to all the commented out code.

iimpulse commented on src/app/analysis/snps-indels/snp-indel-details/snp-indel-details.component.ts:
```diff
@@ -63,13 +64,13 @@ export class SnpIndelDetailsComponent implements OnInit {
 
     openGeneDialog() {
-      console.log("open gene dialog");
+      console.log('open gene dialog');
```
Probably dont need tall these logs either

iimpulse commented on src/app/app.component.ts:
```diff
@@ -56,18 +57,10 @@ export class AppComponent implements OnInit {
-  }
   isMaps(path) {
-    var titlee = this.location.prepareExternalUrl(this.location.path());
+    let titlee = this.location.prepareExternalUrl(this.location.path());
```
variable name typo?

iimpulse commented on src/app/mvar-api/mvar-api.component.ts:
```diff
@@ -95,10 +95,10 @@ public class Login {
 secret = "{username: myusername, password: mypassword}"
-token <- POST (url = "https://bhcape01.jax.org/api/auth/login", 
-               add_headers("API-KEY" = "xxx", "VERSION" = "1", 
+token <- POST (url = "https://bhcape01.jax.org/api/auth/login",
```
might want url to be an env variable

belkassaby replied:
I just removed the whole variables, as there were not used...

iimpulse commented on src/app/mvar-api/mvar-api.component.ts:
```diff
@@ -110,8 +110,8 @@ token <- POST (url = "https://bhcape01.jax.org/api/auth/login",
 
 // Not  yet used
```
Not sure whats up with these guys

iimpulse commented on src/app/notifications/notifications.component.ts:
```diff
@@ -8,29 +8,32 @@ declare var $: any;
   constructor() { }
-  showNotification(from, align){
-      const type = ['','info','success','warning','danger'];
+  showNotification(from, align) {
```
interesting notification handling

belkassaby replied:
I removed the whole component... not used either... some legacy code i think.

iimpulse reviewed (commented):
please tend to some of the stuff i saw

belkassaby committed:
clean up code

belkassaby committed:
update README

belkassaby committed:
Remove decprecated exctractCss and remove unused component

belkassaby committed:
Update stats and add source and assembly info
*comment out "in-progress" section of dashboard

belkassaby committed:
Update stats and add source and assembly info
*add source endpoint for mvar sources
*display sources and assembly in the stat card

belkassaby committed:
Correctly update assembly
